src/app/containers/ArticleTimestamp/shouldLastUpdatedTimestampBeShown.js

```diff
@@ -0,0 +1,33 @@
+import { isLastRelative, isSameDay, isToday } from './helpers';
+
+const hasBeenUpdated = (timeDifferenceMinutes, minutesTolerance) =>
+  timeDifferenceMinutes > minutesTolerance;
+
+const publishedAndUpdatedToday = (firstPublished, lastPublished) =>
+  isToday(firstPublished) && isSameDay(firstPublished, lastPublished);
+
+const publishedAndUpdatedOnSameDayAndNotRelative = (
+  firstPublished,
+  lastPublished,
+) => isSameDay(firstPublished, lastPublished) && !isLastRelative(lastPublished);
+// Could this be reconsidered as publishedAndUpdatedOnDifferentDaysAndLastUpdatedNotRelative
+// !isSameDay
+
+const shouldLastUpdatedTimestampBeShown = ({
+  minutesTolerance,
+  firstPublished,
+  lastPublished,
+}) => {
+  const timeDifferenceMinutes = (lastPublished - firstPublished) / 1000 / 60;
+  return (
+    hasBeenUpdated(timeDifferenceMinutes, minutesTolerance) &&
+    (publishedAndUpdatedToday(firstPublished, lastPublished) ||
+      !publishedAndUpdatedOnSameDayAndNotRelative(
+        firstPublished,
+        lastPublished,
+      ))
+    // Avoid double-negatives and flipping a 'show' condition
```

Comment:
I think you can remove this comment

Reply:
Sorry - this is in progress. There's going to be some refactoring of the double-negative in the condition.

Reply:
All comments removed ;)